server/src/email/templates/special-use-common/application-submitted-confirmation.es6

```diff
@@ -5,7 +5,7 @@ module.exports = (application, defaultApplicationDetails, reviewTime) => {
 
   return {
     to: application.applicantInfoEmailAddress,
-    subject: `Your ${util.camelCaseToRegularForm(application.type)} Open Forest permit application has been submitted for review!`,
+    subject: `Your ${application.type === 'tempOutfitters' ? 'Temporary Outfitting and Guiding' : 'Non-Commercial Group Use'} permit application has been submitted for review!`,
```

Comment:
Per our conversation offline, is there any way we can keep this generic instead of restricting it to two types in a ternary, so it handles any application type that might get added in the future?

Reply:
@briandavidson Hey Brian, I'm going to go back and take another look at this. For some reason I was thinking we decided to roll with this. But, maybe I'm making that up lol. I'll start trying to find a more generic solution until/unless I hear differently from you.

Reply:
I think we had agreed that a generic solution would be best as it would "future proof" this bit of code in case any new application types get added to the system. But what you had did work and it's a bit of a judgement call.

Reply:
Cool cool. I will work towards a generic solution today.